### DISCUSSION TITLE
Example/Docs for using `kfunc`

### DISCUSSION THREAD

**Discussion by ek-nath** [cilium/ebpf#1124] [Q&A]
The recently added support for `kfunc` doesn't have any documentation or examples of how to use it. It'd be nice if there was some examples.

**Comment by ti-mo** [comment]
Good point, although we do have a [testdata case for kfunc](https://github.com/cilium/ebpf/blob/main/testdata/kfunc.c) that is pretty representative of how it's supposed to be used in the wild, if that helps you to get started. I started writing down what needs to be done docs-wise: https://github.com/cilium/ebpf/issues/1152. Subscribe to that one if you'd like to stay posted. Closing this topic for now.
